app/assets/javascripts/admin/addon/routes/admin-web-hooks-edit.js

```diff
@@ -0,0 +1,26 @@
+import DiscourseRoute from "discourse/routes/discourse";
+
+export default DiscourseRoute.extend({
+  serialize(model) {
+    return { web_hook_id: model.id || "new" };
+  },
+
+  model(params) {
+    if (params.web_hook_id === "new") {
+      return this.store.createRecord("web-hook");
+    }
+    return this.store.find("web-hook", params.web_hook_id);
+  },
+
+  setupController(controller, model) {
+    if (model.get("isNew")) {
+      model.set("web_hook_event_types", controller.get("defaultEventTypes"));
+    }
+
+    model.set("category_ids", model.get("category_ids"));
+    model.set("tag_names", model.get("tag_names"));
+    model.set("group_ids", model.get("group_ids"));
```

Comment:
This looks a little odd, why do we need to set these properties to what they already are?